_posts/2020-04-01-deep_learning_33_million_with_few_lines_yaml.md

```diff
@@ -121,7 +121,7 @@ split:
   index_ratio: { train: 0.9, eval: 0.1 }
 ```
 
-Three lines of YAML, but they pack a punch. ZenML will let you [categorize your data before splitting it](https://docs.zenml.io/docs/developer_guide/pipelines_yaml).
+Three lines of YAML, but they pack a punch. ZenML will let you [categorize your data before splitting it](https://docs.zenml.io/).
```

Comment:
Same here, let's just remove the links where we don't have any relevant docs pages anymore.

Reply:
Removing the link altogether.